internal/tool/multi/deployer.go

```diff
@@ -48,27 +48,15 @@ const defaultReplication = 2
 
 // A deployer manages an application deployment.
 type deployer struct {
-	ctx       context.Context
-	ctxCancel context.CancelFunc
-	dep       *protos.Deployment
-	started   time.Time
-	logger    *slog.Logger
-	running   errgroup.Group
-
-	// logSaver processes log entries generated by the weavelet. The entries
```

Comment:
I like that we could delete these functions.

Reply:
Me too! I think when the multiprocess deployer was used for weavertests, we had to pass these in, but now we don't :)